app/views/journeys/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1> ALL THE JOURNEYS </h1>
+
+<ul>
+  <% @journeys.each do |journey| %>
+    <li><%= link_to journey.title, journeys_path(journey)  %></li>
```

Comment:
should be journey_path because we will want to see the show for the choosen journey